R/create_eq.R

```diff
@@ -246,6 +276,18 @@ add_greek.polr <- function(rhs, terms, ...) {
          )
 }
 
+#' @keywords internal
+
+add_greek.clm <- function(rhs, terms, ...) {
+  ifelse(rhs$coef.type == "intercept",
+         anno_greek("alpha",
+                    seq_along(grep("intercept", rhs$coef.type))),
+         anno_greek("beta",
+                    seq_along(grep("location", rhs$coef.type)),
+                    terms)
+  )
+}
```

Comment:
Same thing again. It's really annoying that there are different names for these things because they would just work otherwise. I wonder if we might even want to submit a PR to broom to see if we can get the names consistent.

Reply:
Also, just thinking of other strategies, if we could get the names consistent I wonder if we could just internally change the class of `clm` to `polr`, or maybe added the `polr` class to it. Then we wouldn't even need the additional function(s) at all.

Reply:
Yeah, I'd love to figure out a way to align the `coef.type` column. They're weird in both ologit packages: in `MASS::polr()`, the coefficients are `coefficient` (fine) and the intercepts are `scale` (weird), while in `ordinal::clm()` the coefficients are `location` (weird) and the intercepts are `intercept` (fine). I don't think that's a broom issue, though—broom picks up the internal nomenclature that the packages provide (I think?)

We'll run into similar issues with the `coef.type` later with `lmer()` and multilevel models (and they use completely different names).

Reply:
Ah... yup, that makes sense with broom. I may play around with this later, but it's fairly low priority. One thing we could maybe do is after we tidy it just have a quick function that standardizes the terms and, if it's a `clm` class, switch it to `polr`. Something like that may work.